File: leetcode.com/1929_concatenation_of_array/solution.c
int *getConcatenation(int *nums, int numsSize, int *returnSize)
{
    int *array = malloc(sizeof(int) * numsSize * 2);

    for (int i = 0 ; i < numsSize ; i++)
        array[i] = array[numsSize + i] = nums[i];

    *returnSize = numsSize * 2;
    return array;
}

File: leetcode.com/1929_concatenation_of_array/solution.go
func getConcatenation(nums []int) []int {
    array := make([]int, len(nums)*2)

    for i := 0 ; i < len(nums) ; i++ {
        array[i] = nums[i]
        array[len(nums) + i] = nums[i]
    }

    return array
}
